packages/kotti-checkbox/src/Checkbox.vue

```diff
@@ -17,11 +17,18 @@ export default {
 	name: 'KtCheckbox',
 	props: {
 		label: { default: null, type: String },
+		color: { default: null, type: String },
 		value: { default: false, type: Boolean },
 	},
 	computed: {
-	        hasLabel() {
-			return !(this.label===null || this.label === undefined)
+		hasLabel() {
+			return !(this.label === null || this.label === undefined)
+		},
+		iconColor() {
+			return {
+				backgroundColor: this.color ? this.color : 'none',
+				borderColor: this.color ? this.color : 'none',
```

Comment:
should none be the default?

Reply:
also you need to add an example in the docs

Reply:
example added. None should be the default.

Reply:
Colors should consume the KtTheme by default. Anything else means a shitload of work for any app trying to use consistent colors.

Reply:
Actually there is also a requirement to change the color of the select box. if you knew the design in `AdminPanel` there is colorful checkbox to enable different graph.